Core/clim-core/commands/tables.lisp

```diff
@@ -502,12 +502,15 @@ menu item to see if it is `:menu'."
                          (otherwise nil))))
     ;; Return a literal command, or create a partial command from a
     ;; command-name.
-    (return-from lookup-keystroke-command-item
-      (substitute-numeric-argument-marker
-       (if (symbolp command)
-           (partial-command-from-name command command-table)
-           command)
-       numeric-arg)))
+    (when (or (null *application-frame*)
+              (command-enabled (if (consp command) (command-name command) command)
+                                  *application-frame*))
```

Comment:
command-name already checks whether the comand is a cons or not, so we may skip the check here.

Reply:
Done.